content/en/agent/faq/how-datadog-agent-determines-the-hostname.md

```diff
@@ -64,14 +46,76 @@ When pulling information on your AWS hosts from the [Datadog API][2], the follow
+A single host running in EC2 might have an instance ID (i-abcd1234), a generic hostname provided by EC2 based on the host's IP address (ip-192-0-0-1), and a meaningful hostname provided by an internal DNS server or a config-managed hosts file (myhost.mydomain). Datadog creates aliases for host names when there are multiple uniquely identifiable names for a single host.
 
 The names collected by the Agent (detailed above) are added as aliases for the chosen canonical name.
 
-See a list of all the hosts in your account from the [Infrastructure List][3]. See the list of aliases associated with each host in the Inspect panel, which is accessed by clicking the "Inspect" button while hovering over a host row:
+See the list of all hosts in your account from the [Infrastructure List][4]. The aliases associated with each host are available in the inspect panel, which is accessed by clicking the **Inspect** button while hovering over a host row:
 
 {{< img src="agent/faq/host_aliases.png" alt="Host aliases" responsive="true" >}}
 
+## Agent versions
+
+{{< tabs >}}
+{{% tab "Agent v6" %}}
+
+There are differences in hostname resolution between Agent v5 and Agent v6.
+
+### Linux/MacOS
+
+When upgrading from Agent v5 to Agent v6, there might be a difference in the hostname reported by your Agent. To resolve the system hostname Agent v5 uses the `hostname -f` command while Agent v6 uses the Golang API `os.Hostname()`. On upgrades, the Agent hostname could change from a Fully-Qualified Domain Name (FQDN) to a short hostname, for example:
+
+`sub.domain.tld` --> `sub`
+
+**Note**: Starting from Agent v6.3, a configuration option called `hostname_fqdn` was introduced that allows Agent v6 to have the same behavior as Agent v5. This flag is disabled by default on version 6.3+. See the [example datadog.yaml][1] for enabling this option.
+
+#### Determine if you're affected
+
+Starting with v6.3.0, the Agent logs the warning below if you’re affected by the change:
+```text
+DEPRECATION NOTICE: The agent resolved your hostname as <HOSTNAME>. However in a future version, it will be resolved as <FQDN> by default. To enable the future behavior, please enable the `hostname_fqdn` flag in the configuration.
+```
+
+You are not affected if any of the following are true:
+
+* You are running the Agent in GCE.
+* You are setting the hostname in the [Agent's main configuration][2] file or through the `DD_HOSTNAME` environment variable.
+* You are running the Agent in a container with access to the Docker or Kubernetes API.
+* The hostname output of `cat /proc/sys/kernel/hostname` and `hostname -f` are the same.
+
+#### Recommended action
```

Comment:
It seems that the H4 css layout is broken in tabs, instead of having the purple color, it seems to default to the H3 design

Reply:
We can fix this in a separate PR